igor-core/src/main/java/com/netflix/spinnaker/igor/polling/CommonPollingMonitor.java

```diff
@@ -277,8 +277,8 @@ public Long getLastPoll() {
     return lastPoll.get();
   }
 
-  private boolean isSendingEventsEnabled() {
-    return dynamicConfigService.isEnabled("igor.build.sendEvents", true);
+  private boolean isSendEventsEnabled() {
+    return dynamicConfigService.getConfig(Boolean.class, "spinnaker.build.sendEventsEnabled", true);
```

Comment:
i changed this to `getConfig` vs `isEnabled` to not have the extra `.enabled` tacked on, to make it consistent with the `isPollingEnabled` prop